assets/charts/control-plane/kubelet/container-image/README.md

```diff
@@ -0,0 +1,57 @@
+# Lokomotive Kubelet Image
+
+This image is built on top of the Typhoon Kubelet image. This image includes a wrapper script that runs `isciadm` in the host namespace. This ensures that all the `isciadm` libraries are loaded from the host.
+
+## Updating Image Tag
+
+In this directory run the following command to update the Kubernetes version:
+
+```bash
+export CURRENT_VERSION=
+export NEW_VERSION=
+```
+
+```bash
+sed -i "s|$CURRENT_VERSION|$NEW_VERSION|g" Dockerfile README.md
+```
+
+## Building Image
+
+### x86
+
+```bash
+ARCH=amd64 docker build -t quay.io/kinvolk/kubelet:v1.19.4-amd64 --build-arg ARCH . \
+    && docker push quay.io/kinvolk/kubelet:v1.19.4-amd64
+```
+
+### ARM
+
+```bash
+ARCH=arm64 docker build -t quay.io/kinvolk/kubelet:v1.19.4-arm64 --build-arg ARCH . \
+    && docker push quay.io/kinvolk/kubelet:v1.19.4-arm64
```

Comment:
This is recommended line-breaking, at least in `Dockerfile`.
```suggestion
ARCH=arm64 docker build -t quay.io/kinvolk/kubelet:v1.19.4-arm64 --build-arg ARCH . && \
docker push quay.io/kinvolk/kubelet:v1.19.4-arm64
```

Reply:
These commands are meant to be copy pasted (not for `Dockerfile`), the `\` is going to be overridden and everything will be on one line.